client/src/modules/common/components/Layout/Layout.tsx

```diff
@@ -255,7 +255,7 @@ export const Layout = (): JSX.Element => {
     ];
 
     return (
-        <>
```

Comment:
This should be original, `<>` and `</>` respectively

Reply:
Where? shouldnt it be a div?

Reply:
I'll just approve for now and change it later, won't matter too much anyway.